chanzuckerberg/idseq-web
# Add creation logic to bulk download modal.

# Description

Mostly updates to the model and controller.
Also add error and success messages to the bulk download modal.

<img width="637" alt="Screen Shot 2019-10-10 at 12 58 48 PM" src="https://user-images.githubusercontent.com/837004/66608896-739a5400-eb6c-11e9-8eed-87f3aa22a78c.png">

<img width="651" alt="Screen Shot 2019-10-10 at 12 59 04 PM" src="https://user-images.githubusercontent.com/837004/66608897-75641780-eb6c-11e9-888a-e553b7bc0180.png">

# Tests
* Wrote tests for controller/model changes.
* Verify that bulk download objects are correctly created via the modal.
* Verify that users without the flag cannot access controller methods.
